tox.ini

```diff
@@ -35,6 +37,7 @@ commands =
 setenv = PYTHONPATH = {toxinidir}/src
 deps =
     -rrequirements.txt
+     -r dev_requirements/requirements-type_check.txt
     mypy
     types-pytz
     pandas-stubs
```

Comment:
die händischen dependencies dann bitte noch entfernen, die kommen jetzt ja aus dem ...txt file in der zeile drüber

Reply:
https://github.com/Hochfrequenz/kohlrahbi/pull/201/commits/b42dad8e90f595b7a19db38ef3093d66133e101a